pkg/builder/local_build_executor_test.go

```diff
@@ -691,6 +718,9 @@ func TestLocalBuildExecutorWithWorkingDirectorySuccess(t *testing.T) {
 					"../hello.pic.d",
 					"objects/hello.pic.o",
 				},
+				OutputDirectories: []string{
```

Comment:
I'm not sure that this is a realistic example for a test case. Why would a build action ever need to contain an output path of that shape? Wouldn't that be the same as requesting `"."` (which also happens to be an interesting test case).

Reply:
I chose this because it meant that all of the directories that I would need to know about in the test were ones that already existed as mocks. I'm happy to spend a bit longer creating something more realistic now that I have a bit of feedback though.

Reply:
As it turns out, requesting the output directory `"."` doesn't work currently either, I'll add a fix for that to this PR too.

Reply:
> As it turns out, requesting the output directory `"."` doesn't work currently either, I'll add a fix for that to this PR too.

Latest update solves this.